javagen/src/main/java/com/azure/autorest/template/example/ProtocolExampleWriter.java

```diff
@@ -222,12 +224,14 @@ public ProtocolExampleWriter(ProtocolExample protocolExample) {
                     params.set(i, "Context.NONE");
                 }
             }
-            methodBlock.line(String.format(
-                    "%1$s response = %2$s.%3$s(%4$s);",
-                    method.getReturnValue().getType(),
+            String methodCall = String.format("%s.%s(%s)",
                     clientVarName,
                     method.getName(),
-                    String.join(", ", params)));
+                    String.join(", ", params));
+            if (isTestCode && (method.getType() == ClientMethodType.LongRunningBeginSync || method.getType() == ClientMethodType.LongRunningBeginAsync)) {
+                methodCall = "setPlaybackSyncPollerPollInterval(" + methodCall + ")";
```

Comment:
Should we use `setPlaybackPollerFluxPollInterval` for `LongRunningBeginAsync`?

Reply:
Yes, the method has overload for PollerFlux

Reply:
Their names seem to be different? `setPlaybackPollerFluxPollInterval` vs `setPlaybackSyncPollerPollInterval`
https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java#L351-L378

Reply:
Ah, I see. It is not an overload. Will fix.